### DISCUSSION TITLE
How do I hide these gridlines?

### DISCUSSION THREAD

**Discussion by whjing** [Stellarium/stellarium#3246] [Q&A] [answered]
![image](https://github.com/Stellarium/stellarium/assets/134769085/4ecf35ac-2fa9-4673-9615-fb77fcedb6cc)

After installing on Linux system, these gridlines appear when I open it. How do I remove them? Thank you!

**Comment by 10110111** [comment, marked as answer]
These are not gridlines, these are artifacts resulting from your bad video driver. If you're using `nouveau` on an NVIDIA card, you'll need to replace it with NVIDIA's binary driver, `nvidia`.

**Reply by whjing** [reply]
Thank you so much!!!!! This problem has been troubling me for a long time.
